source/blog/2019-03-08-the-ember-times-issue-88.md

```diff
@@ -12,8 +12,10 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## Only a few days left to complete the [Ember Community Survey](https://emberjs.com/ember-community-survey-2019/)
```

Comment:
The tool that we're using to transfer the blog post to the newsletter e-mail template has issues if not the whole headline is wrapped in the link, therefore I'd suggest to:

```suggestion
## [Only a few days left to complete the Ember Community Survey](https://emberjs.com/ember-community-survey-2019/)
```